### ISSUE TITLE
Super slow greeter interface.

### ISSUE THREAD

**Issue by rRobis** [Antergos/lightdm-webkit2-greeter#65]
So I have  a problem.
I'm using Antergos, and my webkit2-greeter is super slow(interface). When I type in something, it takes like 5 seconds to do that(response delay 5 seconds). The animations have delay. It's like it freezes periodically. The mouse is also slow.

I tried 2 themes and both of them are slow ( Material and Antergos default theme).

I have intel I3 procesor, Intel HD  3000 graphics card, single screen,  8GB of RAM.
How do I fix this?

**Comment by Joshua-Butt** [comment]
i7 5500U 8GB DDR3 ARCH64

also experiencing this issue, 
[LightDM Log FIle](http://pastebin.com/raw/5jSX806e)

**Comment by samueldr** [comment]
I have a workaround for my setup, using archlinux.

File: `/etc/X11/xorg.conf.d/20-intel.conf`

    Section "Device"
            Identifier  "Intel Graphics"
            Driver      "intel"
            Option      "DRI" "2"             # DRI3 is now default 
    EndSection

The issue, for me, appeared when  intel updated to DRI 3. This file is made [from the instructions on the archlinux wiki](https://wiki.archlinux.org/index.php/Intel_graphics#DRI3_issues).

This might help finding out where the issue lies.

* * *

## Info dump

Some glxinfo, just in case:

```
~ ⬤  glxinfo  | grep '^\s*Vendor' -B1 -A10
Extended renderer info (GLX_MESA_query_renderer):
    Vendor: Intel Open Source Technology Center (0x8086)
    Device: Mesa DRI Intel(R) Haswell Mobile  (0xa16)
    Version: 12.0.1
    Accelerated: yes
    Video memory: 1536MB
    Unified memory: yes
    Preferred profile: core (0x1)
    Max core profile version: 3.3
    Max compat profile version: 3.0
    Max GLES1 profile version: 1.1
    Max GLES[23] profile version: 3.0
```

And since the intel graphics hardware is linked to the CPU.

    model name      : Intel(R) Core(TM) i5-4210U CPU @ 1.70GHz

Another model which exhibits the same symptoms, same GLX vendor.

    model name      : Intel(R) Celeron(R) 2955U @ 1.40GHz

Packages

```
xorg-server 1.18.4-1
xf86-video-intel 1:2.99.917+697+g12c14de-1
linux 4.7.2-1
```

**Comment by lots0logs** [comment]
This issue should be addressed by webkit2gtk 2.12.4 (which will be in arch repos soon)

**Comment by lots0logs** [comment]
Duplicate of #61
